src/web/app/pages-logs/logs-page.component.ts

```diff
@@ -29,8 +41,18 @@ interface SearchLogsFormModel {
 interface QueryParams {
   searchFrom: string;
   searchUntil: string;
-  severities: string;
+  severity?: string;
+  minSeverity?: string;
+  logEvent?: string;
   nextPageToken?: string;
+  actionClass?: string;
+  traceId?: string;
+  googleId?: string;
+  regkey?: string;
+  email?: string;
+  sourceLocationFile?: string;
+  sourceLocationFunction?: string;
+  exceptionClass?: string;
```

Comment:
Looks like a lot of similarity between this and `SearchLogsFormModel`

Reply:
I will remove the duplicate fields in `formModel`. The fields will be added to `queryParams` directly instead of going through the `formModel`.

Reply:
Binding data directly to `formModel` leads to some problem. When user changes the filters but do not press the query button, the `queryParams` is still modified. This results in error when user loads more logs as same params are expected. So I will create another interface `advancedFilters` to reduce the duplication instead.